spec/lang/well-formed.md

```diff
@@ -410,6 +440,8 @@ impl PlaceExpr {
             }
             Field { root, field } => {
                 let root = root.check_wf::<T>(locals, prog)?;
+                // TODO(UnsizedTypes): this needs to be relaxed for unsized structs
+                ensure_wf(root.size::<T>().is_sized(), "PlaceExpr::Field: root is unsized")?;
```

Comment:
We're already checking that it is a Tuple or Union. Those are always sized. So this extra check here does not seem necessary?

Reply:
This is indeed unnecessary, particularly since no expect_sized is used in the field expression evaluation.